src/main/kotlin/org/partiql/scribe/targets/redshift/RedshiftCalls.kt

```diff
@@ -1,6 +1,93 @@
 package org.partiql.scribe.targets.redshift
 
+import org.partiql.ast.Ast.exprCall
+import org.partiql.ast.Ast.exprVarRef
+import org.partiql.ast.DatetimeField
+import org.partiql.ast.Identifier
+import org.partiql.ast.expr.Expr
 import org.partiql.scribe.ScribeContext
+import org.partiql.scribe.problems.ScribeProblem
+import org.partiql.scribe.sql.SqlArgs
+import org.partiql.scribe.sql.SqlCallFn
 import org.partiql.scribe.sql.SqlCalls
+import org.partiql.scribe.sql.utils.unquotedStringExpr
 
-public class RedshiftCalls(context: ScribeContext) : SqlCalls(context)
+public open class RedshiftCalls(context: ScribeContext) : SqlCalls(context) {
+    private val listener = context.getProblemListener()
+
+    override val rules: Map<String, SqlCallFn> =
+        super.rules.toMutableMap().apply {
+            this["utcnow"] = ::utcnow
+            // Extensions
+            this["split"] = ::split
+        }
+
+    /**
+     * https://docs.aws.amazon.com/redshift/latest/dg/r_SYSDATE.html
+     */
+    private fun utcnow(args: SqlArgs): Expr {
+        val id = Identifier.regular("sysdate")
+        listener.report(
+            ScribeProblem.simpleInfo(
+                code = ScribeProblem.TRANSLATION_INFO,
+                message = "PartiQL `utcnow()` was replaced by Redshift `SYSDATE`",
+            ),
+        )
```

Comment:
Should translation info also be provided for e.g., the `Redshift does not support x['y'] syntax; replace with x.y` in RedshiftAstToSql.kt?

Reply:
I'll add an info message for this. Previously, we only had info messages for function call translations but this may also be relevant.